src/module/vue/components/buttons/btn-sendmovetochat.vue

```diff
@@ -16,6 +16,7 @@
 
 <script setup lang="ts">
 import { Move } from '../../../features/custommoves'
+import { SFRollMoveDialog } from '../../../helpers/rolldialog-sf'
```

Comment:
couple thoughts:

* once the new IS move data is in 'release' state - will there be much utility in keeping SFRollMoveDialog from the ironsworn version?
* at present there isn't a dialog for doing progress rolls, but i reckon there could be some utility to one. worth renaming it to `*ActionRollDialog`? if nothing else, it'd be more specific/precise and make it immediately apparent to someone viewing the code that progress rolls are intentionally kept out of the dialog

Reply:
* Oh yeah, once we're using Dataforged data for Ironsworn and Starforged both, I'll be able to retire a bunch of code. My intent is that this new rolling engine becomes the One True Engine, and the rest of the rolling infrastructure is retired.
* Including progress rolls in this engine might have some utility, yeah. I'll think on how to represent that.

Reply:
> Including progress rolls in this engine might have some utility, yeah. I'll think on how to represent that.

yeah - there might be some use in having a simple "set the progress score to use and roll". a more elaborate version might also present, say, a dropdown list of all open progress items (perhaps filtered/sorted by the type relevant to the move being rolled, if any)